source/MaterialXView/Glsl/external/tinyexr/TinyEXRImageHandler.cpp

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Image I/O wrapper to use the TinyEXR library. We can always sub this out with a real I/O library later one but this is the smallest usable one for saving/loading floating point images. ImageHandler is the based abstract class which can be used to wrap others.